va/va.h

```diff
@@ -3777,6 +3777,22 @@ VAStatus vaQuerySurfaceError(
     void **error_info
 );
 
+/**
+ * This function blocks during specified timeout (in nanoseconds) until
+ * all pending operations associated with given buf_id have been completed.
+ * If timeout is zero, the function returns immediately.
+ * A common example of usage is synchronization of output encoded buffers, output statistic buffers.
+ * Return values:
+ * VA_STATUS_SUCCESS - all pending operations completed successfully;
+ * VA_STATUS_ERROR_HW_BUSY - operations are still in progress;
```

Comment:
Does `VA_STATUS_ERROR_HW_BUSY` mean the underlying HW hang? How does user know that `timeout_ns` has passed but the underlying HW is still handling the operations?

Reply:
>Does VA_STATUS_ERROR_HW_BUSY mean the underlying HW hang?

I can't say exactly. va.h doesn't describe it in details. If VA_STATUS_ERROR_HW_BUSY really means GPU hang then yes, it's incorrect to return this status here. In this case we would need something like VA_STATUS_ERROR_OPERATION_TIMED_OUT which in fact not an error but rather a warning indicating that not all operations associated with given VABufferID have been completed within timeout. 
What do you think?

Reply:
Addressed by VA_STATUS_ERROR_TIMEDOUT